hack/lib/swagger.sh

```diff
@@ -20,23 +20,22 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# The root of the build/dist directory
-KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE}")/../.." && pwd -P)"
-
 # Generates types_swagger_doc_generated file for the given group version.
 # $1: Name of the group version
 # $2: Path to the directory where types.go for that group version exists. This
 # is the directory where the file will be generated.
 kube::swagger::gen_types_swagger_doc() {
   local group_version=$1
   local gv_dir=$2
-  local TMPFILE="${TMPDIR:-/tmp}/types_swagger_doc_generated.$(date +%s).go"
+  local TMPFILE
+  TMPFILE="${TMPDIR:-/tmp}/types_swagger_doc_generated.$(date +%s).go"
```

Comment:
non-trivial assignment should be separate from declaration in order to not swallow the error